fplus-lib/src/core/application/file.rs

```diff
@@ -272,6 +272,7 @@ pub enum AppState {
     StartSignDatacap,
     Granted,
     TotalDatacapReached,
+    DeclineApplication,
```

Comment:
```suggestion
    Declined,
```
`Decline` is a verb, we want an adjective here.
No need to add `Application` in variant name, when this entire type already is called `AppState`, so it's obvious it's about an application